docs/build/wasm/from-zero-to-ink-hero/manic-minter/manic-e2e.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 4
+---
+
+# ManicMinter e2e Test
+In this chapter we will write e2e tests for the ManicMinter contract. The e2e tests will be written in Rust using the ink! e2e framework. The e2e tests will be executed on a local substrate node.
+Just like in previous chapter we will not include complete code from the contract to keep it short and focused on the e2e tests.
+## Import Crates
+To start with, we will import the following crates:
```

Comment:
Can you precise where to add this new `mod` ?

Reply:
done